src/config/quantTypes.json

```diff
@@ -384,6 +384,58 @@
       ]
     }
   },
+  "library10XVDJ": {
+    "key": "library10XVDJ",
+    "name": "Library - 10X VDJ - Tapestation",
+    "wellType": "TS10XVDJ",
+    "parse": {
+      "delimiter": ",",
+      "rowDelimiter": ["\r\n", "\r", "\n", "\r\r\n"],
+      "from": 2,
+      "columns": [
+        "filename",
+        "wellId",
+        "sampleDescription",
+        "fromBasePair",
+        "toBasePair",
+        "averageSizeBasePairs",
+        "concentration",
+        "regionMolarity",
+        "percentageOfTotal",
+        "regionComment"
+      ]
+    },
+    "metadata": {},
+    "conversion": {
+      "factors": {
+        "dilution": 10
+      },
+      "expression": "(dilution)",
+      "decimalPlaces": 3
+    },
+    "qcResults": {
+      "key": "molarity",
```

Comment:
I don't know if this is the right key to use.  The data was exported to my local SequenceScape install when I tried to upload the same file (after creating a plate) but the web interface of SequenceScape didn't show the values sent in.  I assume the concentration value was supposed to appear here?  Maybe it was the plate type I created in SequenceScape that explains why it was the wrong key?  Some other plate types maybe show molarity?  I don't know enough about these systems to know whether this is OK or not.

![image](https://user-images.githubusercontent.com/1999241/129939411-c4e48dc4-4ef6-40dd-9e9c-7b4e32467b52.png)

Reply:
Maybe you have to have the delayed working running locally to process it? But I'm sorry I'm not sure about the plate type, @JamesGlover might know more?

Reply:
Concentration is normally measured in grams per Litre (i.e. a mass per volume, usually nanograms per microlitre [ng/ul]) or in number of Moles per Litre (molarity, number of molecules per volume, usually nanomolar [nM]). Molarity would need to be calculated, you should be able to see these calculations in Quanthub where other molarity keys are used. Possibly the Tapestation does this calculation for you, don't know.
Also for this story the user seems to be talking about 'region molarity (nmol/L)' so that may be some new way of recording concentration that we don't normally handle. Need to discuss with user I think. I can join if you like.